launcher/container_runner.go

```diff
@@ -472,10 +472,10 @@ defaultRetryPolicy retries as follows:
 Given the following arguments, the retry sequence will be:
 
 	RetryInterval = 60 sec
-	RandomizationFactor = 0.5
+	JitterPercentage = 50
 	Multiplier = 2
-	MaxInterval = 3600 sec
-	MaxElapsedTime = 0 (never stops retrying)
+	WithCappedDuration = 3600 sec
+	MaxRetries = 8
```

Comment:
Should we keep the unlimited retry?

Reply:
my concern is the launcher will hang forever with unlimited retry if `refreshToken` keeps returning error. We might want the launcher exit in this case.

Reply:
nvm, this PR will use the original library, so we still keep the unlimited retry.